.github/workflows/nightly-test-integrations-1.18.x.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       runs-on: ${{ needs.setup.outputs.compute-large }}
       repository-name: ${{ github.repository }}
       uploaded-binary-name: 'consul-bin'
-      branch-name: "release/1.17.x"
+      branch-name: "release/1.18.x"
```

Comment:
This was causing failures like the following in container logs:
![image](https://github.com/user-attachments/assets/3b55c626-0d1a-4c16-a7f7-af4ed795462c)

Reply:
Good find!